### ISSUE TITLE
edge_weight is overriden in FeaturePropagation

### ISSUE THREAD

**Issue by bgmello** [twitter-research/feature-propagation#1]
In the method `get_propagation_matrix` of the class `FeaturePropagation` the variable `edge_weight` is overriden in the following code:

```
edge_weight = edge_weight if edge_weight else torch.ones(edge_index.shape[1]).to(edge_index.device)
edge_index, edge_weight = get_symmetrically_normalized_adjacency(edge_index, n_nodes=n_nodes)
```

Since in the function `get_symmetrically_normalized_adjacency` all weights are defined as 1, this code essentially makes the graph unweighted.

**Comment by emalgorithm** [comment]
Thanks, solved! The idea is to always use the symmetrically normalized adjacency, so edge weights are not really supported at the moment. I've removed them from the code.
